examples/ensure-name-segment/prefix/README.md

```diff
@@ -0,0 +1,36 @@
+# ensure-name-substring: Prefix Example
+
```

Comment:
I think adding a note about it being experimental here for all the docs (`Help`, `Prefix`, `Suffix`, `Advanced`)

```
Note: This is an alpha function and we are actively seeking feedback on the function config syntax and behavior. If you have suggestion or feedback, please file a issue [here](https://github.com/GoogleContainerTools/kpt/issues/new/choose).
```